runtime/development/src/lib.rs

```diff
@@ -1720,10 +1786,10 @@ impl pallet_block_rewards::Config for Runtime {
 impl pallet_transfer_allowlist::Config for Runtime {
 	type CurrencyId = FilterCurrency;
 	type Deposit = AllowanceDeposit<Fees>;
-	type HoldId = HoldId;
-	type Location = Location;
+	type Location = RestrictedTransferLocation;
 	type ReserveCurrency = Balances;
 	type RuntimeEvent = RuntimeEvent;
+	type RuntimeHoldReason = RuntimeHoldReason;
```

Comment:
⚠️  we probably need to migrate previous holds done by TransferAllowlist to the new `HoldReason` value